Sources/SwiftGRPCNIO/HTTP1ToRawGRPCServerCodec.swift

```diff
@@ -58,7 +65,9 @@ extension HTTP1ToRawGRPCServerCodec: ChannelInboundHandler {
 
       state = .expectingCompressedFlag
       buffer = ctx.channel.allocator.buffer(capacity: 5)
-
+      if let contentTypeHeader = requestHead.headers["Content-Type"].first {
```

Comment:
Should `"Content-Type"` be lowercased? (I think at least in the HTTP2 spec all headers are lowercase.)

Reply:
the headers object doesn't care for the case of the field, lowercased to be consistent.